src/WebJobs.Script.WebHost/Resources/app_offline.htm

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml" >
+<head>
+    <title>Down for Maintenance</title>
+    <style type="text/css">
+        * {
+            margin: 0;
+            padding: 0;
+        }
+
+        a {
+            text-decoration: underline;
+            color: #FF381A;
+        }
+
+        a:hover {
+            text-decoration: none;
+        }
+
+        body {
+            line-height: 1.75em;
+            background: #fff;
+            font-size: 11.5pt;
+            color: #5A6466;
+        }
+
+        body,input {
+            font-family: Kreon, serif;
+        }
+
+        br.clearfix {
+            clear: both;
+        }
+
+        h1,h2,h3,h4 {
+            text-transform: uppercase;
+            font-weight: normal;
+        }
+
+        h2 {
+            font-size: 1.5em;
+        }
+
+        h2,h3,h4 {
+            font-family: "Open Sans", sans-serif;
+            color: #2A3436;
+            margin-bottom: 1em;
+        }
+
+        h3 {
+            font-size: 1.25em;
+        }
+
+        h4 {
+            font-size: 1em;
+        }
+
+        img.alignleft {
+            float: left;
+            margin: 5px 30px 20px 0;
+        }
+
+        img.aligntop {
+            margin: 5px 0 20px 0;
+        }
+
+        p {
+            margin-bottom: 1.5em;
+        }
+
+        a {
+            color: #2A3436;
+        }
+
+        .box {
+            margin: 0 0 250px 0;
+        }
+
+        img.alignleft {
+            float: left;
+            margin: 5px 30px 30px 0;
+            border-radius: 6px;
+        }
+
+        img.aligntop {
+            margin: 5px 0 20px 0;
+            border-radius: 6px;
+        }
+
+        #content {
+            padding: 0;
+            width: 815px;
+            margin: 0 0 0 0px;
+        }
+
+
+        #page {
+            margin: 0;
+            position: relative;
+            width: 900px;
+            padding: 20px 40px 0 40px;
+        }
+
+
+        #wrapper {
+            width: 978px;
+            position: relative;
+            background: #FFF;
+            margin: 0 auto 0 auto;
+            box-shadow: 0px 0px 300px 0px rgba(0,0,0,0.15);
+            border: solid 1px #82A7AD;
+            
+        }
+
+        
+    </style>
+</head>
+<body>
+    <div>&nbsp;</div>
+    <div id="wrapper">
+        
+        <div id="page">
+            <div id="content">
+
+                <div class="box">
+                    <h2>Azure Functions is Down for maintenance</h2>
```

Comment:
Might be helpful to have a clearer message so it doesn't confuse users into thinking the the "Azure Functions Service" is down. Perhaps something like "This Azure Functions site is down for maintenance" ?

Reply:
Fixed